ERROR - 2023-11-03 16:58:58 --> ErrorException: pg_query(): Query failed: ERROR:  no existe la relación «registro.fn_reporte3»
LINE 2: FROM &quot;registro&quot;.&quot;fn_reporte3&quot;
             ^ in C:\xampp\htdocs\cursos_ceta\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'pg_query(): Que...', 'C:\\xampp\\htdocs...', 135)
#1 C:\xampp\htdocs\cursos_ceta\system\Database\Postgre\Connection.php(135): pg_query(Object(PgSql\Connection), 'SELECT *\nFROM "...')
#2 C:\xampp\htdocs\cursos_ceta\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute('SELECT *\nFROM "...')
#3 C:\xampp\htdocs\cursos_ceta\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM "...')
#4 C:\xampp\htdocs\cursos_ceta\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM "...', Array, false)
#5 C:\xampp\htdocs\cursos_ceta\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\cursos_ceta\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\cursos_ceta\app\Controllers\Reportes.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(832): App\Controllers\Reportes->index()
#9 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Reportes))
#10 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\cursos_ceta\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-11-03 16:58:58 --> Call to a member function getResult() on bool
#0 C:\xampp\htdocs\cursos_ceta\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#1 C:\xampp\htdocs\cursos_ceta\app\Controllers\Reportes.php(19): CodeIgniter\BaseModel->findAll()
#2 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(832): App\Controllers\Reportes->index()
#3 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Reportes))
#4 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\cursos_ceta\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-11-03 17:00:08 --> Too few arguments to function CodeIgniter\Database\BaseBuilder::where(), 0 passed in C:\xampp\htdocs\cursos_ceta\system\Model.php on line 728 and at least 1 expected
#0 C:\xampp\htdocs\cursos_ceta\system\Model.php(728): CodeIgniter\Database\BaseBuilder->where()
#1 C:\xampp\htdocs\cursos_ceta\app\Controllers\Reportes.php(19): CodeIgniter\Model->__call('where', Array)
#2 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(832): App\Controllers\Reportes->index()
#3 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Reportes))
#4 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\cursos_ceta\public\index.php(37): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-11-03 17:02:25 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\cursos_ceta\app\Views\reportes\reportes.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 30)
#1 C:\xampp\htdocs\cursos_ceta\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\cursos_ceta\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\cursos_ceta\system\Common.php(1089): CodeIgniter\View\View->render('reportes/report...', Array, true)
#4 C:\xampp\htdocs\cursos_ceta\app\Controllers\Reportes.php(23): view('reportes/report...', Array)
#5 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(832): App\Controllers\Reportes->index()
#6 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Reportes))
#7 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\cursos_ceta\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2023-11-03 17:06:13 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\cursos_ceta\app\Views\reportes\reportes.php(30): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Trying to acces...', 'C:\\xampp\\htdocs...', 30)
#1 C:\xampp\htdocs\cursos_ceta\system\View\View.php(210): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\cursos_ceta\system\View\View.php(213): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 C:\xampp\htdocs\cursos_ceta\system\Common.php(1089): CodeIgniter\View\View->render('reportes/report...', Array, true)
#4 C:\xampp\htdocs\cursos_ceta\app\Controllers\Reportes.php(23): view('reportes/report...', Array)
#5 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(832): App\Controllers\Reportes->index()
#6 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(419): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Reportes))
#7 C:\xampp\htdocs\cursos_ceta\system\CodeIgniter.php(326): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\cursos_ceta\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
